dumpling/export/dump.go

```diff
@@ -706,6 +737,65 @@ func (d *Dumper) sequentialDumpTable(tctx *tcontext.Context, conn *BaseConn, met
 	return d.dumpWholeTableDirectly(tctx, meta, taskChan, "", orderByClause, 0, 1)
 }
 
+func (d *Dumper) handleBuffer(totalChunk int) {
+	if !d.conf.SQLConcurrent {
+		return
+	}
+	set := make(map[int]*bytes.Buffer)
+	cond := sync.NewCond(new(sync.Mutex))
+	finished := make(chan struct{})
+	go func() {
+		for {
+			select {
+			case <-finished:
+				return
+			case r := <-d.bufChan:
+				cond.L.Lock()
+				set[r.idx] = r.buf
+				cond.Signal()
+				cond.L.Unlock()
+			}
+		}
+	}()
+	d.wg.Go(
```

Comment:
seems no one is waiting for finish writing

Reply:
Outside writing group is waiting. https://github.com/pingcap/tidb/blob/master/dumpling/export/dump.go#L328